doc/neps/nep-0016-zero-rank-arrarys.rst

```diff
@@ -0,0 +1,238 @@
+=========================
+NEP 16 — Zero Rank Arrays
```

Comment:
let's save NEP 16 for https://github.com/numpy/numpy/pull/10706

Reply:
ok, so this becomes NEP 27.